site/Gemfile.lock

```diff
@@ -50,7 +50,7 @@ DEPENDENCIES
   minima (~> 2.0)
 
 RUBY VERSION
-   ruby 2.4.1p111
+   ruby 2.0.0p648
 
 BUNDLED WITH
-   1.15.1
+   1.15.3
```

Comment:
Please revert changes to the Gemfile.log. Only changes needed for the implementation you are adding should be in this PR.